library/ssl_msg.c

```diff
@@ -499,6 +499,82 @@ static void ssl_extract_add_data_from_record( unsigned char* add_data,
 
     *add_data_len = cur - add_data;
 }
+#endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
+static void ssl_extract_add_data_from_record( unsigned char* add_data,
```

Comment:
Why was this change made? The functions are mostly the same across TLS 1.2 and TLS 1.3, so IMO it makes sense to share the code and add a guard for the section that differs across 1.2 and 1.3?